apps/server/src/services/rundown-service/rundownCache.ts

```diff
@@ -117,50 +121,53 @@ export function generate(
 
     if (isOntimeEvent(currentEntry)) {
       currentEntry.delay = 0;
+      currentEntry.gap = 0;
 
       // 1. handle links - mutates updatedEvent
       handleLink(i, initialRundown, currentEntry, links);
 
       // 2. handle custom fields - mutates updatedEvent
       handleCustomField(customFields, customFieldChangelog, currentEntry, assignedCustomFields);
 
+      if (lastEntry) {
+        const lastEntryCrossedMidnight = lastEntry.timeStart + lastEntry.duration > dayInMs;
+        const isNextDay = checkIsNextDay(lastEntry.timeStart, currentEntry.timeStart, lastEntry.duration);
+        totalDays += isNextDay || lastEntryCrossedMidnight ? 1 : 0; //TODO: should we count in days or millis
```

Comment:
@cpvalente what do you think of this?
It feels a bit more easy to reason about and human to stor it as number of days, but most (if not all) of the places we end up using the data we multiply it with our `dayInMs` constant anyway...